internal/pkg/platform/platform_matcher.go

```diff
@@ -115,11 +115,22 @@ func getCPUVariant(os string, arch string) string {
 	return ""
 }
 
+// compatibility contains, for a specified architecture, a list of known variants, in the
+// order from most capable (most restrictive) to least capable (most compatible).
+// Architectures that don’t have variants should not have an entry here.
 var compatibility = map[string][]string{
 	"arm":   {"v7", "v6", "v5"},
 	"arm64": {"v8"},
```

Comment:
I think arm64 runs arm as well, right?

Reply:
From what I hear, that is CPU-dependent, not an architecture requirement.

Reply:
Correct.

Though there is "arm/v8" as well - the ARMv8 32bit instructions.

Reply:
`arm/v8` is the one referenced in #3626, 64 bits kernel on 32 bits userland.

Reply:
https://github.com/cri-o/cri-o/issues/3626 should work with this PR (arm/v8 will now use the image for arm/""); but with the list above, arm/v8 would not use a arm/v7 image.

I can add v8 to the 32-bit ARM list above in this PR as well (causing the arm/v8 compatibility list to be arm/{v8,v7,v6,v5,""} ), if there is consensus on that.

Reply:
Yes. ARMv8 defines both a 32bit ISA (superset of ARMv7) and a 64bit ISA.

Some CPUs only support one of them, so `arm/v8` has to be treated differently from `arm64/v8`.

Reply:
Thanks, added.